src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/assets/baselines/MsftToSbSdk.diff

```diff
@@ -545,7 +563,18 @@ index ------------
 -./sdk/x.y.z/zh-Hant/Microsoft.VisualStudio.Coverage.IO.resources.dll
  ./sdk/x.y.z/zh-Hant/MSBuild.resources.dll
  ./sdk/x.y.z/zh-Hant/System.CommandLine.resources.dll
- ./sdk/x.y.z/zh-Hant/vstest.console.resources.dll
  ./shared/
- ./shared/Microsoft.AspNetCore.App/x.y.z/
-
+@@ ------------ @@
+ ./templates/
+ ./templates/x.y.z/
+ ./templates/x.y.z/microsoft.dotnet.common.itemtemplates.x.y.z.nupkg
+-./templates/x.y.z/microsoft.dotnet.common.projecttemplates.x.y.z.0.116.nupkg
++./templates/x.y.z/microsoft.dotnet.common.projecttemplates.x.y.z.0.115.nupkg
```

Comment:
I see there are some version numbers here.  If these change regularly, consider logging a follow up issue to abstract/filter these out to eliminate having to update the baselines regularly.